pkg/util/slices/slices_test.go

```diff
@@ -57,42 +57,47 @@ func TestToRefMap(t *testing.T) {
 
 func TestToCmpNoOrder(t *testing.T) {
 	cases := map[string]struct {
-		sliceA      []int
-		sliceB      []int
-		expectedRes bool
+		sliceA []int
+		sliceB []int
+		want   bool
 	}{
 		"correct_single_occurence_value": {
```

Comment:
nit: I would suggest dropping the "_" in test case names. Also, maybe replace "correct" / "incorrect" with "equal multisets", and "unequal multisets".